user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -824,5 +834,230 @@ function testCasesForPolicyTable:Restore_preloaded_pt()
 		commonPreconditions:RestoreFile(pt_fileName)
 	end
 end	
-	
+
+-- The funcion will be used when PTU is triggered. 
+-- 1. It is assumed that notification is recevied: EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+-- 2. It is assumed that request/response is received: EXPECT_HMICALL("BasicCommunication.PolicyUpdate", 
+-- 3. Function will use default endpoints
+function testCasesForPolicyTable:flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY(app_id, device_id, hmi_app_id, ptu_file_path, ptu_file_name)
+	function Test:TestStep_Flow_PTU_SUCCEESS_EXTERNAL_PROPRIETARY()
```

Comment:
@Itileda Update for new functions, for old one I didn't update because don't know where and how are used.
Additional tests will be also added, comments can be seen in google sheet.